integration_tests/handler/user_test.go

```diff
@@ -181,6 +181,9 @@ func TestUpdateUser(t *testing.T) {
 				res = testutils.DoRequest(t, e, http.MethodPatch, e.URL(api.User.UpdateUser, tt.userID), &tt.reqBody)
 				testutils.AssertResponse(t, tt.statusCode, tt.want, res)
 				// Get updated response & Assert
+				if *tt.reqBody.Check == false {
+					user.RealName = ""
+				}
```

Comment:
`tt.reqBody.Check`がnilのときにnil pointerでpanicするのでnilじゃないことを確かめる必要がありますね

```suggestion
				if tt.reqBody.Check != nil && *tt.reqBody.Check == false {
					user.RealName = ""
				}
```